### PR TITLE
Fix required assemblies quantity limit

### PR DESCRIPTION
**Pull request by gustavopvasconcellos**
#### What problem is this solving?

Definição do bug

It's possible to insert more than the limit "extras" assemblies by changing the input (on the example the limit is 3)

#### How to test it?

- https://qamcdbrdeliveryio.myvtex.com/
- set the postal code: 04552-000 
- search for "Big Mac" and access the product page
- try to add "20" of some extras.

[Fixed Workspace](https://limit--qamcdbrdeliveryio.myvtex.com/big-mac/p)

#### Screenshots or example usage:

https://prnt.sc/z83p5i

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+
+- Add verification to prevent context changes quantity to more than the assembly limit
```
Comment (icazevedo) [inline CHANGELOG.md:11]:
```suggestion
### Fixed
- Add verification to prevent context changes quantity to more than the assembly limit
```

**Review by icazevedo — APPROVED** [review]
LGTM

File: react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx [hunk 2]
```diff
@@ -133,7 +133,7 @@ const ProductAssemblyOptionItemCustomize: FC<Props> = ({
         {...buttonProps}
       >
         <div
-          className={`${handles.productAssemblyOptionItemCustomize__label} c-action-primary t-action`}
+          className={`${handles.productAssemblyOptionItemCustomizeLabel} c-action-primary t-action`}
```
Comment (igorbrasileiro) [inline react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx:136]: Remember to update this `productAssemblyOptionItemCustomize__label` css handle at README.

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]
Update README file changing the correctly css handle,  
from: `productAssemblyOptionItemCustomize__label`  
to: `productAssemblyOptionItemCustomizeLabel`

File: react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx (hunk 2 above)
Comment (gustavopvasconcellos) [reply react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx]: @igorbrasileiro its done ;)

File: docs/README.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `productAssemblyGroupNameRow` |
 | `productAssemblyGroupRequiredTag` |
 | `productAssemblyOptionItemCustomize` |
-| `productAssemblyOptionItemCustomize__label` |
+| `productAssemblyOptionItemCustomizeLabel` |
```
Comment (victorhmp) [inline docs/README.md:127]: This is a breaking change. Stores could be using the `productAssemblyOptionItemCustomize__label`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Add verification to prevent context changes quantity to more than the assembly limit
```
Comment (victorhmp) [inline CHANGELOG.md:10]:
```suggestion
- Users were able to add unlimited quantities of a certain assembly option. Now the limit configured for each option is enforced.
```

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 3 above)
Comment (icazevedo) [reply docs/README.md:127]: @victorhmp actually it's not! There's something weird with this branch: it doesn't share the same history with master, but it is letting us merge 🤔 

I've fixed this handle [on this PR](https://github.com/vtex-apps/product-customizer/pull/80). It was preventing the app from building (other previous was merged incorrectly).

What I suggest @gustavopvasconcellos to do is to rebase your branch on the vtex-apps/master branch. That way, this fix will no longer be necessary.

Comment (victorhmp) [reply docs/README.md:127]: Awesome!! Nice catch @icazevedo 😄

**Pull request closed without merging**